values.yaml

```diff
@@ -0,0 +1,30 @@
+redis:
+  enabled: true
+  auth:
+    enabled: false
+  architecture: standalone
+  master:
+    persistence:
+      enabled: true
+      size: 1Gi  # Adjust the size as needed
+
+onetimesecret:
+  image: 
+    repository: onetimesecret/onetimesecret
+    tag: latest
+  service:
+    type: LoadBalancer
```

Comment:
Not sure why it was a blocker but the onetimesecret pod wouldn't start in my environment until i set this to ClusterIP. I am running it locally via k3ds

Reply:
@blacktig3r any thoughts on this one?